libs/tdb/Makefile

```diff
@@ -45,9 +45,45 @@ define Build/InstallDev
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/* $(1)/usr/lib/
 endef
 
+CONFIGURE_ARGS = \
+		--target=$(GNU_TARGET_NAME) \
+		--build=$(GNU_HOST_NAME) \
+		--cross-compile \
+		--cross-answers="$(PKG_BUILD_DIR)/cache.txt" \
+		--program-prefix="" \
+		--prefix=$(CONFIGURE_PREFIX) \
+		--exec-prefix=$(CONFIGURE_PREFIX) \
+		--bindir=$(CONFIGURE_PREFIX)/bin \
+		--sbindir=$(CONFIGURE_PREFIX)/sbin \
+		--libexecdir=$(CONFIGURE_PREFIX)/lib \
+		--sysconfdir=/etc \
+		--datadir=$(CONFIGURE_PREFIX)/share \
+		--localstatedir=/var \
+		--mandir=$(CONFIGURE_PREFIX)/man \
+		--infodir=$(CONFIGURE_PREFIX)/info \
+		$(DISABLE_IPV6)
+
+CONFIGURE_ARGS += \
+		--disable-python \
+		--disable-rpath \
+		--disable-rpath-install \
+		--disable-rpath-private-install
+
+CONFIGURE_VARS += \
+		CPP="$(TARGET_CROSS)cpp"
+
+define Build/Configure
+	$(CP) ./files/tdb.cache.txt $(PKG_BUILD_DIR)/cache.txt
+	echo -e "\nChecking uname sysname type: \"$(VERSION_DIST)\"" \
+	"\nChecking uname release type: \"$(LINUX_VERSION)\"" \
+	"\nChecking uname machine type: \"$(ARCH)\"" \
+	"\nChecking uname version type: \"$(VERSION_DIST) Linux-$(LINUX_VERSION) $(shell date +%Y-%m-%d)\"\n" >> $(PKG_BUILD_DIR)/cache.txt;
```

Comment:
Don't need a semi if there's not another command on the same line.  Also, please indent your continuation lines.